evap/staff/templates/staff_program_merge.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'staff_base.html' %}
+
+{% load evaluation_filters %}
+
+{% block breadcrumb %}
+    {{ block.super }}
+    <li class="breadcrumb-item"><a href="{% url 'staff:program_index' %}">{% translate 'Programs' %}</a></li>
+    <li class="breadcrumb-item">{% translate 'Merge programs' %}</li>
+{% endblock %}
+
+{% block content %}
+    {{ block.super }}
+    <h3>{% translate 'Merge programs' %}</h3>
+
+    <div class="card mb-3">
+        <div class="card-body">
+            <p>
+                {% blocktranslate %}The following courses currently have the program <strong>{{ other_instance }}</strong> and will
+                get the program <strong>{{ main_instance }}</strong> once you merge the programs. The program <strong>
+                {{ other_instance }}</strong> will be deleted after that.{% endblocktranslate %}
```

Comment:
I _think_ this will show up more nicely in the translation file when using `blocktranslate trimmed` like with "Do you really want to delete the semester ..."